components/Hero.tsx

```diff
@@ -100,6 +102,14 @@ const Hero = ({
           </a>
         )}
       </div>
+
+      <br />
+
+      <div className='cta-container'>
+          {props.showTicketAvailability && (
+            <TicketCounter />
```

Comment:
I like this and I think we can already merge but can we also try to have the tickets on the right side of the hero?

Reply:
I merge this, we try to move this component in another pr